src/components/Table/DelegatesTable.tsx

```diff
@@ -0,0 +1,136 @@
+    console.log('delegate', delegate)
+    const candidateInfo = Candidates.find((deleg) => deleg.address.toLowerCase() === delegate.address.toLowerCase())
+    setSelectedCandidate({ ...delegate, ...candidateInfo! })
+  }
+
+  const updateSort = (newSortingKey: DelegateKey) => {
+    if (newSortingKey !== sortingKey) {
+      setSortingKey(newSortingKey)
+    } else {
+      changeSort()
+    }
+  }
+
+  const buildTable = () => {
+    return (
+      <Table unstackable basic="very" className="DelegatesTable">
+        <Table.Header className="DelegatesTable__Header">
+          <Table.Row>
+            <Table.HeaderCell
+              className={TokenList.join(['DelegatesTable__CandidateNameHeader', isMobile && 'DelegatesTable__Sticky'])}
+            >
+              {t('page.home.dao_delegates.candidate_name')}
+            </Table.HeaderCell>
+            {isMobile && <Table.HeaderCell className="DelegatesTable__ShadowBox DelegatesTable__ShadowBoxHeader" />}
+            <Table.HeaderCell
+              className={TokenList.join([isMobile && 'DelegatesTable__PaddedColumn'])}
+              onClick={() => updateSort('lastVoted')}
+            >
+              <span>
+                {t('page.home.dao_delegates.last_voted')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'lastVoted'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('timesVoted')}>
+              <span>
+                {t('page.home.dao_delegates.times_voted')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'timesVoted'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('pickedBy')}>
+              <span>
+                {t('page.home.dao_delegates.picked_by')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'pickedBy'} />
+              </span>
+            </Table.HeaderCell>
+            <Table.HeaderCell onClick={() => updateSort('totalVP')}>
+              <span>
+                {t('page.home.dao_delegates.total_vp')}
+                <Sort descending={isDescendingSort} selected={sortingKey === 'totalVP'} />
+              </span>
+            </Table.HeaderCell>
+            {!isMobile && <Table.HeaderCell />}
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {delegatesSorted.slice(0, DISPLAYED_CANDIDATES).map((delegate, index) => (
+            <DelegateRow
+              key={delegate.address + 'row' + index}
+              delegate={delegate}
+              sticky={isMobile}
+              onDelegateSelected={handleOnDelegateClick}
+            />
+          ))}
+        </Table.Body>
+      </Table>
+    )
+  }
+
+  return (
+    <>
+      {isMobile && (
+        <div className="DelegatesTable__Wrapper">
+          <div className="DelegatesTable__Scroller">{buildTable()}</div>
```

Comment:
It'd be great to have a Table component to reuse instead of a function. 

And is it possible to "media-query" these two classes (__Wrapper and __Scroller) so you don't need to render conditionally here?